viewer/packages/cad-geometry-loaders/src/CadModelUpdateHandler.ts

```diff
@@ -208,10 +207,12 @@ function createDetermineSectorsInput([settings, camera, clipping, models]: [
   ClippingInput,
   CadNode[]
 ]): DetermineSectorsPayload {
+  const prioritizedAreas = models.map(model => model.prioritizedAreas).flat();
```

Comment:
```suggestion
  const prioritizedAreas = models.flatMap(model => model.prioritizedAreas);
```
This is the same, right?